components/partials/modals/bridge/index.vue

```diff
@@ -211,6 +212,17 @@ export default Vue.extend({
   },
 
   computed: {
+    maxDecimals(): number {
+      const { form } = this
+
+      // This check makes sure that the max amount of decimals is never greater than the amount of decimal places being shown in the balance.
```

Comment:
Let's replace the comment using a description const variable for the condition check, for example:
```js
const tokenDecimalLessThenDefault = `form.token.decimals < UI_DEFAULT_DISPLAY_DECIMALS`
```